integreat_cms/cms/models/events/event.py

```diff
@@ -29,6 +29,18 @@
     from ..users.user import User
 
 
+class CouldNotBeCopied(Exception):
+    """
+    Exception for events that can't be copied
+    """
+
+    def __init__(
+        self,
+        message: str = "Event couldn't be copied because it's from a external calendar",
+    ) -> None:
+        super().__init__(message)
```

Comment:
```suggestion
```

I think its fine to completely get rid of this part, because this exception is not supposed to ever be printed (Because it should always get catched)